types/anime-quotes-api/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for anime-quotes-api 1.2
+// Project: https://github.com/anjannair/anime-quotes-api
+// Definitions by: 1chiSensei <https://github.com/1chiSensei>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class Quote {
+	constructor();
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/anime-quotes-api/index.d.ts;
  Line: 7;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.